azure/deploy-to-azure/azure-integration/azuredeploy.json

```diff
@@ -34,6 +34,10 @@
                 "description": "Subscriptions list to monitor"
             }
         },
+        "subscriptionCount": {
+            "type": "int",
+            "defaultValue": "[length(parameters('subscriptionsList'))]"
```

Comment:
What's the meaning of `default` in this case? I would think this would always be the value, not just by default

Reply:
Parameter fields in ARM templates renders as a text box and expects a value. As the length of the `subscriptionCount` is inferred from `subscriptionList`, we set the default value and do not expose this field.